jobs/openshift/cluster/integreatly/install/Jenkinsfile

```diff
@@ -77,6 +77,29 @@ node('cirhos_rhel7') {
         if (params.dryRun) {
             println("Would call ansibleTower Integreatly Install Workflow template with the following installer options: ${installerOptions}")
         } else {
+            def additionalParams = ""
+            if(params.additionalParameters != null) {
+                additionalParams = params.additionalParameters.split(',')
+            }
+
+            def extraVars = """---
+            tower_environment: dev 
+            cluster_name: ${installerOptions.clusterName}
+            openshift_master_public_url: ${installerOptions.openshiftMasterUrl}
+            openshift_username: ${installerOptions.clusterAdminUsername}
+            openshift_password: ${installerOptions.clusterAdminPassword}
+            integreatly_project_install_scm_url: ${installerOptions.installationGitUrl}
+            integreatly_project_install_scm_branch: ${installerOptions.installationGitBranch}
+            eval_seed_users_count: ${installerOptions.userCount}
+            eval_self_signed_certs: '${installerOptions.selfSignedCerts}'
+            integreatly_inventory_source_aws_credentials: ${installerOptions.awsCredentials}"""
+
+            // append additional parameters    
+            for(String param : additionalParams) {
+                extraVars = extraVars + "\n            " + param.trim()
```

Comment:
This is extra ugly, but I don't know how to do that better. I tried to use map but I had some issues with serialization (not sure why)